scripts/development/create_databases.sh

```
@@ -22,5 +23,22 @@ if [ -f $gamesdb ]; then
 # create db, tables
+
+# CAVEAT 1 (players): `id INT PRIMARY_KEY` matches prod but is actually is a
+# bug. `PRIMARY_KEY` (with underscore) is not a keyword, so `id` is a plain
+# nullable INT column, not primary key and not rowid alias. Kept as-is to match
+# prod. See CAVEAT 2 for how this doesn't affect production checks for the PK.
+
+# CAVEAT 2 (players) : missing prod player indexes Confirmed that there are
+# additional prod indexes: - idx_players_id (to compensate for CAVEAT 1) -
+# idx_players_name Left alone in development DB creation, for now.
+
 echo "CREATE TABLE players (id INT PRIMARY_KEY, name VARCHAR(20), password VARCHAR(50), email VARCHAR(50), rating real default 1000, boost real default 750, ratedgames int default 0, maxrating real default 1000, ratingage real default 0, ratingbase int default 0, unrated int default 0, isbot int default 0, fatigue text default '{}', is_admin int default 0, is_mod int default 0, is_gagged int default 0, is_banned int default 0, participation_rating int default 1000);" | sqlite3 $playersdb
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)